static/temp/foods.json

```diff
@@ -0,0 +1,250 @@
+[
+  {
+    "name": "Banana ripe, slightly ripe, raw",
+    "categoryName": "Fruit and Fruit Juices",
+    "categoryIcon": "EggAltOutlinedIcon",
+    "overview": {
+      "info": "This is a cool food",
```

Comment:
Cool Food has 3 items, hence, this could be an optional object like following

```text
"coolFood": {
 "icon": "IconName",
 "text": "This is a cool food",
 "moreInfo": {
   "description": "Cool Food Meals are a simple and delicious way to help the planet.",
   "linkText": "Learn More",
   "link": "https://coolfood.org/consumer/"
 }
}
```

Since not all foods will be eligible for Cool Food, this object will only be available for foods that are eligible.

Reply:
done